extensions/wikia/ImageReview/ImageCountGetter.class.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+class ImageCountGetter extends WikiaModel {
+
+	const QUESTIONABLE = 'questionable';
+	const REJECTED     = 'rejected';
+	const UNREVIEWED   = 'unreviewed';
+	const INVALID      = 'invalid';
+
+
+	public function getImageCounts( $reviewerId ) {
+		return $this->mergeCounts(
+			$this->getCountOfAllUnassignedImages(),
+			$this->getCountOfThisUsersInProgressImages( $reviewerId )
+		);
+	}
+
+	/**
+	 * Get count of all images which have yet to be assigned a reviewer. This is for
+	 * both images that haven't gone through the first screening where they are marked
+	 * as questionable, rejected, or approved by the first level reviewers, and for
+	 * those images marked rejected or questionable which haven't been assigned to a staff
+	 * member to make the final call.
+	 * @return bool|mixed
+	 * @throws Exception
+	 */
+	private function getCountOfAllUnassignedImages() {
+		return ( new WikiaSQL() )
+			->SELECT()
+			->FIELD( 'state' )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "state" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)